content/actions/index.md

```diff
@@ -55,4 +55,11 @@ body = '''
 Add a GitHub status link directly to a CircleCI artifact.
 '''
 
+[[item]]
+type = 'card'
+header = '[MeeseeksDev / Lumberbot (App)](https://github.com/scientific-python/MeeseeksDev)'
```

Comment:
@Carreau has moved the bot to the Scientific Python org, but we're going to rename it later once we have validated everything works. We can do a follow up PR once that has been done.

Reply:
c.f. https://github.com/scientific-python/MeeseeksDev/issues/99#issuecomment-2151144959